_locales/en/messages.json

```diff
@@ -683,7 +683,7 @@
         "message": "Disarm motors after set delay [seconds] (Requires MOTOR_STOP feature)"
     },
     "configurationDisarmKillSwitch": {
-        "message": "Disarm motors regardless of throttle value (When arming via AUX channel)"
+        "message": "Arm/Disarm motors regardless of throttle value when using AUX channel"
```

Comment:
@piepluquad:

- You don't have to open a new PR. Just update the branch that this PR is based on, and push, and the PR will automatically update to reflect the new tip of your branch;
- I think for clarity, it is better if the help text specifically mentions **arm**, since the name of the mode switch is **ARM**. If not, noob users will be left searching for a **DISARM** mode switch.